### PR TITLE
3.0.0-preview2

### PR DESCRIPTION
**Pull request by Tommo-L**

### CODE REVIEW

File: neo-cli/CLI/Helper.cs [hunk 1]
```diff
@@ -19,5 +29,51 @@ public static bool IsYes(this string input)
 
             return input == "yes" || input == "y";
         }
+
+        public static JObject ToJson(this StackItem item)
```
Comment (shargon) [inline neo-cli/CLI/Helper.cs:33]: Why don't move this to `neo` it's duplicated in neo-modules, and `neo-node`

Comment (Tommo-L) [reply neo-cli/CLI/Helper.cs:33]: It was fixed in https://github.com/neo-project/neo/pull/1569, but not merged in neo-preview2.

**Comment by Tommo-L** [comment]
Neo-preview2 has been released.

File: neo-cli/CLI/Helper.cs (hunk 1 above)
Comment (mutrofin517) [reply neo-cli/CLI/Helper.cs:33]: AQtk8ckQwnZTowVgs6zgCZEJyUQyM1z3ew

**Pull request closed without merging**